src/mcts/search.cc

```diff
@@ -191,12 +196,23 @@ int64_t Search::GetTimeSinceStart() const {
       .count();
 }
 
+// Root is depth 0, i.e. even depth.
+float Search::GetDrawScore(bool is_odd_depth) const {
+  return (is_odd_depth ? params_.GetOpponentDrawScore()
+                       : params_.GetSidetomoveDrawScore()) +
+         (is_odd_depth == played_history_.IsBlackToMove()
+              ? params_.GetWhiteDrawDelta()
+              : params_.GetBlackDrawDelta());
+}
```

Comment:
Should we catch drawscores above 1.0 or below -1.0 in this function and cap them to 1.0 resp. -1.0? The result of the summation could exceed these values.

Reply:
It seems we work with such Qs just fine? (due to fpu they often happen to be so).
I'm a bit hesitant to put unnecessary checks for something that will be run for every visit, but probably it's negligible, so if there is any moderately valid reason for it, I can add it.

Reply:
While it works fine with how we handle Q in general, having a draw score outside of `[-1.0, 1.0]` will place draws either above wins or below losses, therefore not maintaining the order of moves. With `w_1 >= w_2, w_1 + d_1 >= w_2 + d_2` one should always get `Q_1 >= Q_2` in my opinion.

Reply:
Ok, then I can just add a check the parameters that 
`max(abs(white_draw_score), abs(black_draw_score)) + max(abs(sidetomove_draw_score), abs(opponent_draw_score)) <= 1`.

Then D multiplier will never be more than 1, and neither will Q.
(as `Q = W - L + draw_multiplier*D` and `W+L+D=1`)

Reply:
Yes, that check sounds like a sensible solution and completely addresses my concerns.

Reply:
Done.